app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, html: {'data-turbo' => "false"}, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true %>
+  </div>
+
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
```

Comment:
- [ ] I would suggest following a design for the sign up page to look more professional.

### Your design 

![Screenshot from 2022-03-25 03-41-55](https://user-images.githubusercontent.com/22434660/160019393-6bdd1f40-94ea-49a0-bd1d-af5ade06d8ac.png)

### Given design
![Screenshot from 2022-03-25 03-48-18](https://user-images.githubusercontent.com/22434660/160019488-86dc7943-3465-4f17-99a7-dfee3728f1a3.png)

Reply:
This issue is well resolved! ✔️